x-pack/plugin/downsample/qa/mixed-cluster/build.gradle

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import org.elasticsearch.gradle.Version
+import org.elasticsearch.gradle.internal.info.BuildParams
+import org.elasticsearch.gradle.testclusters.StandaloneRestIntegTestTask
+
+apply plugin: 'elasticsearch.internal-yaml-rest-test'
+apply plugin: 'elasticsearch.internal-java-rest-test'
+apply plugin: 'elasticsearch.internal-test-artifact'
+apply plugin: 'elasticsearch.bwc-test'
+
+
+dependencies {
+  testImplementation project(path: ':test:test-clusters')
+  javaRestTestImplementation project(path: xpackModule('rollup'))
+  yamlRestTestImplementation project(path: xpackModule('rollup'))
+}
+
+restResources {
+  restApi {
+    include '_common', 'bulk', 'cluster', 'indices', 'search', 'ingest.put_pipeline', 'ingest.delete_pipeline'
+  }
+}
+
+artifacts {
```

Comment:
This block is only required if we plan to include these tests in the rest-resources artifact used by the clients team for their test. Given this is specifically for BWC testing I don't think it's applicable so we can remove this block.

Reply:
++, I removed this in https://github.com/elastic/elasticsearch/pull/106878/commits/dfa59e9ba6666b4918a843cd2abd4f5e2ec08a74